pkgs/by-name/zu/zug/package.nix

```diff
@@ -0,0 +1,29 @@
+{ callPackage, stdenv, lib, pkgs,
+  fetchFromGitHub,
+  cmake, boost
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "zug";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "arximboldi";
+    repo = "zug";
+    rev = "v${version}";
+    hash = "sha256-7xTMDhPIx1I1PiYNanGUsK8pdrWuemMWM7BW+NQs2BQ=";
+  };
+  buildInputs = [
+    cmake
```

Comment:
cmake should likely be a nativeBuildInputs

Reply:
cmake has been moved.